backend/src/main/kotlin/cz/loono/backend/api/controller/AccountController.kt

```diff
@@ -68,7 +68,7 @@ class AccountController(
         val aux = UserAuxiliary(
             nickname = patch.nickname,
             preferredEmail = patch.preferredEmail,
-            sex = patch.sex?.name,
+            sex = patch.sex!!.name,
```

Comment:
We should definitely rewrite it as we discussed to allow partial updates where `null` would be considered as a invalid values, since all the values in the `patch` are mandatory

Reply:
These changes will be part of the next refactoring - it's in the DTO's definitions.